<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabs</class>
 <widget class="QTabWidget" name="tabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="repl_tab">
   <attribute name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/UI/syntagma/icons/console.png</normaloff>:/UI/syntagma/icons/console.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Console</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QWebView" name="console">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="bRestart">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/UI/syntagma/icons/restart.png</normaloff>:/UI/syntagma/icons/restart.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="bSend">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/UI/syntagma/icons/send.png</normaloff>:/UI/syntagma/icons/send.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="bHistory">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/UI/syntagma/icons/history.png</normaloff>:/UI/syntagma/icons/history.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="input"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/UI/syntagma/icons/docs.png</normaloff>:/UI/syntagma/icons/docs.png</iconset>
   </attribute>
   <attribute name="title">
    <string>Documentation</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QToolButton" name="bBack">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/UI/syntagma/icons/back.png</normaloff>:/UI/syntagma/icons/back.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="bHome">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/UI/syntagma/icons/home.png</normaloff>:/UI/syntagma/icons/home.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="bForward">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/UI/syntagma/icons/fwd.png</normaloff>:/UI/syntagma/icons/fwd.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QWebView" name="docview">
      <property name="url">
       <url>
        <string>file:///mnt/D/Code/Hylas/docs/html/index.html</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
